changelog/README.rst

```diff
@@ -0,0 +1,39 @@
+*********************
+Manging the Changelog
+*********************
+
+This directory contains "newsfragments" which are short files that contain a
+small **ReST**-formatted text that will be added to the next ``CHANGELOG``.
```

Comment:
Can we please use `ReStructuredText` term? https://en.wikipedia.org/wiki/ReStructuredText
Google search for `ReST` gives a lot of misleading results. Even better: maybe link it to do SphinX docs page.